react-frontend/src/components/StyleComponents/pageContent.js

```diff
@@ -50,3 +50,25 @@ export const ContentBlock = styled.div`
     padding-bottom: 60px;
   }
 `;
+
+export const CovidBannerStyle = styled.div`
+  background-color: #355992;
+  color: #fff;
+  position: fixed;
+  width: 100%;
+  z-index: 1001;
+`;
+
+export const EmptyBannerStyle = styled.div`
+    height:0%;
+    padding: 0;
+    z-index=0;
+`;
+
+export const CovidButtonStyle = styled.a`
```

Comment:
I would follow the same best practice code climate uses for css here and try to keep the components in alphabetical order.  I only realized recently how much that can help.

Reply:
yeah, unfortunately styled components do not support auto formatting to ensure alphabetical order. I will do it manually.